bin/kano-os-loader

```diff
@@ -20,7 +20,9 @@ rm -rf /var/tmp/kano-splash
 mount /proc -t proc /proc
 
 # Launch splash animation in the background, see kano-splash.service
+# Then return the filesystem to Read-Only mode.
 kano-splash-daemonize boot "-t 30 /usr/share/kano-desktop/animations/bootup"
+/bin/mount -o remount,ro /dev/mmcblk0p2 /
```

Comment:
Maybe it would be better to change kano-splash to put its lock files in `/run/lock` so they are removed on next boot? That way we don't need to remount the filesystem at all.

Reply:
Totally agree, but `/run/lock` is not mounted yet, not even `/proc` or anything else,
because we are above `init`. It is prepared by `mountall.sh` init script a lot later.

Reply:
Ah, okay.
👍